modules/hmmer/hmmbuild/main.nf

```diff
@@ -0,0 +1,35 @@
+process HMMER_HMMBUILD {
+    tag '$meta.id'
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::hmmer=3.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmmer:3.3.2--h87f3376_2':
+        'quay.io/biocontainers/hmmer:3.3.2--h1b792b2_1' }"
+
+    input:
+    tuple val(meta), path(alignment)
+
+    output:
+    tuple val(meta), path("*.hmm"), emit: hmm
+    path "versions.yml",            emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    """
+    hmmbuild \\
+        $args \\
+        --cpu $task.cpus \\
+        ${meta.id}.hmm \\
```

Comment:
I would explicitly add the `  -o <f> : direct summary output to file <f>, not stdout` as such a log may be interesting to some people (for multiQC maybe? I don't know what the contents looks like admittedly)

Reply:
Done, thanks for the suggestion.